File: atividade_3/salario.c
#include <stdio.h>

int main (){
    double sal = 0;
    double aumento = 0;
    double novoSal = 0;


    printf("Qual o salario atual:\n");
    scanf("%lf", &sal);
    printf("Qual o aumento:\n");
    scanf("%lf", &aumento);

    novoSal = sal + (sal * (aumento/100));
    

    printf("O salario com o aumento de %.2lf\n", aumento);
    printf("é R$%.2lf\n",novoSal);
}

File: atividade_3/salario.py
salario = float(input("Digite o valor do salário: "))
aumento_percentual = float(input("Digite a porcentagem de aumento: "))


valor_aumento = salario * (aumento_percentual / 100)

novo_salario = salario + valor_aumento

print(f"O valor do aumento é: {valor_aumento}")
print(f"O novo salário é: {novo_salário}")
